lib/search.js

```diff
@@ -17,28 +17,47 @@ module.exports = new class LibSearch {
     }
 
     optimizeQueryTerms(input) {
+        // parse
+        let normalize = this.normalizeProt();
+        input.imdbid = input.imdbid && parseInt(input.imdbid.toString().replace('tt', ''), 10)
+        input.season = input.season && parseInt(input.season)

```

Comment:
we don't know how OS server handles undefined values, and you're also sending useless bytes of information in the POST request, as the json will contain `'season':'undefined'` and use precious brandwidth